### DISCUSSION TITLE
VidCoder - How to enable Intel QSV in VidCoder?

### DISCUSSION THREAD

**Discussion by mpek** [RandomEngy/VidCoder#1296] [General]
On an older laptop Intel QSV is being used earlier with different software, but QSV Encoder is not availale in VidCoder, same with Handbrake. It is not being offered in Codec pulldown menu. 
Is this intentional or do I need a special driver for VidCoder/Handbrake that the QSV would be initialized?

**Comment by RandomEngy** [comment]
The [docs for HandBrake](https://handbrake.fr/docs/en/latest/technical/video-qsv.html) say "Intel Skylake (6th Generation Core) CPU or later with Intel HD, Iris Xe or Arc graphics."

The HandBrake people will probably be able to answer better than I; VidCoder just uses the HandBrake encoding API.
